analysis/import/UnderstandImporter/build.gradle

```diff
@@ -14,6 +14,7 @@ dependencies {
 
     runtime group: 'org.slf4j', name: 'slf4j-simple', version: slf4j_version
 
+    testImplementation('org.junit.jupiter:junit-jupiter:5.6.0')
```

Comment:
Why do we need junit5 if we run the tests using spek2?
Isn't it enough to just include spek2?